src/components/projects/UserProjectsTable.vue

```diff
@@ -209,6 +224,22 @@ export default {
         console.warn(err)
       })
     },
+    inspectProject (project_name, created_at) {
+      const request_params = {
+        project_name,
+        created_at
+      }
+      const project_endpoint = this.$store.state.api_endpoints.projects_endpoint + '/get-project'
+      axios.post(project_endpoint, request_params).then(response => {
+        const project_loader = {
+          project: response.data
+        }
+        this.$store.dispatch('project_params/saveProjectDraft')
```

Comment:
When the new CreateProjectForm is rendered with project data, it re-writes the current project data in the store, so we need to first save off anything that has been entered before we destroy it, so we can load it back later!